test/python/transpiler/test_minimum_point.py

```diff
@@ -0,0 +1,289 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""MinimumPoint pass testing"""
+
+import math
+
+from qiskit.transpiler.passes import MinimumPoint
+from qiskit.dagcircuit import DAGCircuit
+from qiskit.test import QiskitTestCase
+
+
+class TestMinimumPointtPass(QiskitTestCase):
+    """Tests for MinimumPoint pass."""
+
+    def test_minimum_point_reached_fixed_point_single_field(self):
+        """Test a fixed point is reached with a single field."""
+
+        min_pass = MinimumPoint(["depth"], prefix="test")
+        dag = DAGCircuit()
+        min_pass.property_set["depth"] = 42
+        min_pass.run(dag)
+        # After first iteration state is only initialized but not populated
+        state = min_pass.property_set["test_minimum_point_state"]
+        self.assertEqual(state.since, 0)
+        self.assertEqual((math.inf,), state.score)
+        self.assertIsNone(state.dag)
+        self.assertIsNone(min_pass.property_set["test_min_point"])
+
+        # Second iteration
+        min_pass.run(dag)
+        # After second iteration the state is initialized
+        state = min_pass.property_set["test_minimum_point_state"]
+        self.assertEqual(state.since, 1)
+        self.assertEqual(state.score, (42,))
+        self.assertIsNone(min_pass.property_set["test_min_point"])
```

Comment:
I think `test_min_point` should be `test_minimum_point` here? (And everywhere)

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9612/commits/f889fbd11fcabe597688c34ba756bdc6d69f582b